src/global/global_helpers.R

```diff
@@ -1491,6 +1502,8 @@ ms_conversions <- function(d,
         NH3 = 'N',
         SiO2 = 'Si',
         SO4 = 'S',
+        PO4 = 'P',
+        NO3_NO2 = 'NN',
```

Comment:
@spencerrhea 
We should consider NO3_NO2 to be NO3 for the purposes of conversion. Nitrite (NO2) is much rarer in the environment, the molar mass of an assortment of NO3_NO2 is essentially that of NO3. And we don't want to use 'NN' here because that implies there are two Ns for every molecule of NO3 _or_ NO2.